### PR TITLE
Check if we changed access FROM restricted.

### PR DESCRIPTION
**Pull request by HelenaEksler**
#15

### CODE REVIEW

**Comment by HelenaEksler** [comment]
Content privacy is also changing when we change group privacy from restricted to private.

File: pluggable_node_access.module [hunk 1]
```diff
@@ -239,7 +239,7 @@ function pluggable_node_access_set_og_group_content_grants(OgMembership $og_memb
 function pluggable_node_access_og_access_invoke_node_access_acquire_grants_alter(&$context) {
   foreach (pluggable_node_access_get_pluggable_node_access_plugins() as $plugin_name => $plugin) {
     $handler = pluggable_node_access_get_pluggable_node_access_handler($plugin_name, $context['entity']);
-    if ($handler->checkForNodeAccessChange()) {
+    if ($handler->checkForNodeAccessChange() || $context['entity']->pluggable_node_access != $context['entity']->original->pluggable_node_access) {
```
Comment (amitaibu) [inline pluggable_node_access.module]: Add comment - this line is hard to follow

File: pluggable_node_access.module [hunk 2]
```diff
@@ -239,7 +239,10 @@ function pluggable_node_access_set_og_group_content_grants(OgMembership $og_memb
 function pluggable_node_access_og_access_invoke_node_access_acquire_grants_alter(&$context) {
   foreach (pluggable_node_access_get_pluggable_node_access_plugins() as $plugin_name => $plugin) {
     $handler = pluggable_node_access_get_pluggable_node_access_handler($plugin_name, $context['entity']);
-    if ($handler->checkForNodeAccessChange()) {
+    // Check if entity access was changed.
+    // If we are removing restricted access - checkForNodeAccessChange() returns
+    // false, so we should check if the editing entity had restricted access.
+    if ($handler->checkForNodeAccessChange() || $context['entity']->pluggable_node_access != $context['entity']->original->pluggable_node_access) {
```
Comment (amitaibu) [inline pluggable_node_access.module]: so it seems we are hardcoding the `pluggable_node_access` field name - but it's wrong.

**Comment by HelenaEksler** [comment]
Please add a test

**Comment by HelenaEksler** [comment]
@amitaibu This PR is related to #21 and also to PR in OG module https://github.com/Gizra/og/pull/25.

**Comment by HelenaEksler** [comment]
Version of the OG module was changed to the 7.x-2.x-dev. 
@amitaibu PR is ready.

**Comment by HelenaEksler** [comment]
Merged manually.
